src/community/features-templating/features-templating-web/src/main/java/org/geoserver/featurestemplating/web/TemplatePreviewPanel.java

```diff
@@ -381,10 +397,29 @@ private Catalog getCatalog() {
         return (Catalog) GeoServerExtensions.bean("catalog");
     }
 
-    private CloseableHttpClient buildHttpClient() {
+    CloseableHttpClient buildHttpClient() {
         RequestConfig clientConfig =
                 RequestConfig.custom().setConnectTimeout(60000).setSocketTimeout(60000).build();
-        return HttpClientBuilder.create().setDefaultRequestConfig(clientConfig).build();
+        CookieStore cookieStore = new BasicCookieStore();
+        RequestAttributes attributes = RequestContextHolder.getRequestAttributes();
+        if (attributes != null) {
+            BasicClientCookie cookie =
+                    new BasicClientCookie("JSESSIONID", attributes.getSessionId());
+            HttpServletRequest request = request();
+            cookie.setPath(request.getContextPath());
+            Calendar calendar =
```

Comment:
Move the comment the line above, so that it does not get wrapped?

Reply:
fixed